pkg/plugin/plugin.go

```diff
@@ -183,6 +185,45 @@ func Uninstall(name string) error {
 	return os.RemoveAll(pluginDir)
 }
 
+// Information gets the information about an installed plugin
+func Information(name string) (string, error) {
+	pluginDir := filepath.Join(dir(), name)
+
+	if _, err := os.Stat(pluginDir); err != nil {
+		if os.IsNotExist(err) {
+			return fmt.Sprintf("Could not find a plugin called '%s', did you install it?", name), nil
+		}
+		return "", err
+	}
+
+	plugin, err := loadMetadata(pluginDir)
+	if err != nil {
+		return "", err
+	}
+
+	return fmt.Sprintf(`
+Plugin: %s
+  Description: %s
+  Version:     %s
+  Usage:       %s
+
+`, plugin.Name, plugin.Description, plugin.Version, plugin.Usage), nil
+}
+
+// List gets a list of all installed plugins
+func List() (string, error) {
+	plugins, err := LoadAll()
```

Comment:
I prefer displaying an empty list when no plugin is installed. The error is shown currently.

```
$ ./trivy plugin list
2021-12-12T11:23:00.642+0900    FATAL   plugin information display error: failed to read /Users/teppei/.trivy/plugins: open /Users/teppei/.trivy/plugins: no such file or directory
```

There are 2 approaches
  1. Check if the plugin directory exists and returns no error when the dir doesn't exist
  2. When the plugin dir doesn't exist, we will create it in `LoadAll()`.

Reply:
I've gone with Oprion 1:

```bash
./trivy plugin list                                                                                      
There are no plugins installed
```